citrine-scala-style.xml

```diff
@@ -106,7 +106,7 @@
  </check>
  <check level="error" class="org.scalastyle.scalariform.MethodNamesChecker" enabled="true">
   <parameters>
-   <parameter name="regex"><![CDATA[^[a-z][A-Za-z0-9]*$]]></parameter>
+   <parameter name="regex"><![CDATA[^[a-z][A-Za-z0-9_]*$]]></parameter>
```

Comment:
Could we just ignore these when we need to inter-operate with `python_names`?  The command is `// scalastyle:ignore`

Reply:
That's fine with me. I lean slightly toward just allowing it in style rules but care littlest amount possible.